webapps-schema/src/main/resources/schema/elasticsearch/create/index/authorization.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
TBH I have no enough knowledge to judge this. But from what I can tell looks ok to me.